apps/web/pages/settings/my-account/profile.tsx

```diff
@@ -488,6 +502,12 @@ const ProfileForm = ({
         <div className="mt-6">
           <TextField label={t("email")} hint={t("change_email_hint")} {...formMethods.register("email")} />
         </div>
+        <div className="mt-6">
+          <TextField label="LinkedIn Link" {...formMethods.register("linkedinLink")} />
```

Comment:
I tried using t{("linkedinLink")} and added a text in common.json English but it was now working. I would require help here